service/device/registry/internal/src/test/java/org/eclipse/kapua/service/device/registry/common/DeviceValidationTest.java

```diff
@@ -22,13 +22,15 @@
 import org.eclipse.kapua.service.device.registry.DeviceCreator;
 import org.eclipse.kapua.service.device.registry.internal.DeviceCreatorImpl;
 import org.junit.Test;
+import org.junit.Ignore;
 import org.mockito.Mockito;
 
 import static java.math.BigInteger.ONE;
 import static org.junit.Assert.fail;
 import static org.mockito.Matchers.any;
 import static org.mockito.Mockito.mock;
 
+@Ignore
```

Comment:
Hi @andrejn-ct ,

I forgot that if we are running Cucumber tests, jUnit tests must be ignored, otherwise they fails.

Regards, 

_Alberto_